README.md

```diff
@@ -173,12 +178,12 @@ From here, you should be able to do 2 things:
 
 See the data that is being published to the file:
 ```
-$ tail -f /tmp/published
+$ tail -f /tmp/snap-published-mock-file.log
 ```
 
 Or actually tap into the data that snap is collecting:
```

Comment:
@mjbrender Switched it to this since we aren't specifically watching the ID.

"Or actually tap into the data that snap is collecting using the task ID to watch the task:"